tests/foreman/ui/test_rhc.py

```diff
@@ -0,0 +1,193 @@
+"""Test class for RH Cloud connector - rhc
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RHCloud-CloudConnector
+
+:Assignee: jpathan
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.datafactory import gen_string
+from robottelo.helpers import file_downloader
+from robottelo.logging import logger
+
+
+@pytest.fixture(scope='module')
+def subscribe_rhc_satellite(clean_rhsm, module_target_sat):
```

Comment:
yes we already have a fixture for this [subscription.py#L16](https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/subscription.py#L16)

Reply:
Not sure why you removed it and not using the global fixtures as well `module_subscribe_satellite` ! Did you merge this code anywhere ! The last comment is not clear !

Reply:
I removed it because it was not required anymore. It's covered in `fixture_enable_rhc_repos` instead. I am not using [module_subscribe_satellite](https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/subscription.py#L16) because it contains code that unregisters satellite after test execution.